### PR TITLE
chore: enable cd to auto release

### PR DESCRIPTION
**Pull request by Abingcbc**
Enable cd to auto release

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1 @@
+buildPlugin()
```
Comment (NotMyFault) [inline Jenkinsfile:1]:
```suggestion
buildPlugin(useContainerAgent: true)
```

**Review by NotMyFault — COMMENTED** [review]
Looks good so far, but you forgot to create the `extensions.xml`.
Add `.mvn/extensions.xml` with the following content:
```xml
<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
  <extension>
    <groupId>io.jenkins.tools.incrementals</groupId>
    <artifactId>git-changelist-maven-extension</artifactId>
    <version>1.4</version>
  </extension>
</extensions>
```

File: Jenkinsfile (hunk 1 above)
Comment (NotMyFault) [reply Jenkinsfile:1]: Container agents are generally preferred.

Comment (Abingcbc) [reply Jenkinsfile]: Done

File: pom.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
     <properties>
         <revision>1.0</revision>
```
Comment (NotMyFault) [inline pom.xml:28]:
```suggestion
```

**Review by NotMyFault — COMMENTED** [review]
That is everything

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**